### ISSUE TITLE
New AWS account deployment fails to enable AWS Identity Center

### ISSUE THREAD

**Issue by troyfactor4** [awslabs/landing-zone-accelerator-on-aws#71]
**Describe the bug**
On a fresh deploy of LZA 1.3.0 on a new Organization the deploy will fail. The only workaround is to enable AWS Identity Center manually and deploy again.

**To Reproduce**

1. Open a new account.
2. Follow steps to create required LZA accounts and OUs
3. Enable a config that uses AWS Identity Center
4. Deploy the config without manually enabling AWS Identity Center
5. The pipeline fails at the Operations stage with the following message:
```
AWSAccelerator-OperationsStack-111111111111-us-east-1 | 23/39 | 8:24:20 PM | CREATE_FAILED        | AWS::CloudFormation::CustomResource | IdentityCenterGetInstanceId/getIdentityCenter/Default (IdentityCenterGetInstanceIdgetIdentityCenter6DD43AEE) Received response status [FAILED] from custom resource. Message returned: User: arn:aws:sts::111111111111:assumed-role/AWSAccelerator-Operations-IdentityCenterGetInstanc-1EHZOAZIG3WSD/AWSAccelerator-Operations-IdentityCenterGetInstanc-QNbJM4xRuVU3 is not authorized to perform: sso:ListInstances
```
Enabling AWS Identity Center manually and redeploying fixed the issue. All of the Pipeline and Lambda logs show that LZA was able to successfully turn on AWS Identity center as well as successfully set the delegate admin to the proper account.

We set up a few sample users/roles/permissions in the accounts in question and used the AWS CLI to test. Sure enough, running `list-instances` was met with the same permissions error as the pipeline.

```
$ AWS_REGION=us-east-1 aws sso-admin list-instances

An error occurred (AccessDeniedException) when calling the ListInstances operation: User: arn:aws:iam::111111111111:user/testingssoDELETEME is not authorized to perform: sso:ListInstances
```

We believe that even though APIs claim to enable AWS Identity center and will return some values that correlate to this, it is actually not fully enabled.

**Expected behavior**
AWS LZA should be able to turn on AWS Identity Center through APIs and deploy successfully on fresh accounts where AWS Identity Center has not yet been manually enabled.

**Please complete the following information about the solution:**

- [ ] Version: 1.3.0
- [ ] Region: us-east-1
- [ ] Was the solution modified from the version published on this repository? No
- [ ] If the answer to the previous question was yes, are the changes available on GitHub? N/A
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses? N/A
- [ ] Were there any errors in the CloudWatch Logs? Yes, noted above.

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**

We have all logs, and can assist in debugging.

**Comment by farmerbean** [comment]
QQ - does it get through security stack OK? We saw a logfile that it enabled idc .. but we assume because of the delegation from security-config.yaml
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/17742334/220151488-fe76d917-3e61-4a0d-9e3f-cffa6e245c53.png">

**Comment by farmerbean** [comment]
the documentation for iam idc is poor - e.g. https://catalog.workshops.aws/landing-zone-accelerator/en-US/lza-administrators/initial-customization/aws-sso-setup/aws-sso-overview

.. which doesn't tell you which file the config is IN..

**Comment by troyfactor4** [comment]
@farmerbean - we've found the best documentation is the source code itself :joy: 

Also here: https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.IamConfig.html

**Comment by farmerbean** [comment]
Yeah we're also enjoying the "3 layers of documentation" here! 😂 

Still not sure where to put that config, so need to open another ticket.

**Comment by farmerbean** [comment]
Actually have just worked it out from your post @troyfactor4 - you have to look at each class and work out the breadcrumb trail from the documentation..

<img width="970" alt="image" src="https://user-images.githubusercontent.com/17742334/220167651-78197489-d2bc-4811-9b0c-1af8831f46af.png">

**Comment by Athanasii** [comment]
Hello @troyfactor4, thank you for filing an issue with the Landing Zone Accelerator team!

The Landing Zone Accelerator team has talked about this internally and decided not to implement this feature. Here is our reasoning:

The AWS Identity Center API does not yet support enabling AWS Identity Center programmatically.

Closing this issue, please feel free to follow up if you have questions on this issue just make sure to ping me so that I'll see it.
